### ISSUE TITLE
PDF delivery note

### ISSUE THREAD

**Issue by browningjp** [Jon-Salmon/Durham-Hires#56]
with space for signatures!

**Comment by Jon-Salmon** [comment]
I agree this should be added. What kind of info do you think it should be on it? So far i've got:

 - Date printed
 - List of items and prices
 - Space for a signature

**Comment by CJEBond** [comment]
Previously, I provided a pro forma invoice when equipment was collected (see attached). At Trevs, we also have our T's & C's form completed in duplicate, which is where the signature goes.
[Pro Forma Invoice.docx](https://github.com/Jon-Salmon/Durham-Hires/files/2055500/Pro.Forma.Invoice.docx)
[Terms & Conditions 27-10-15.docx](https://github.com/Jon-Salmon/Durham-Hires/files/2055501/Terms.Conditions.27-10-15.docx)

**Comment by browningjp** [comment]
The terms and conditions could be quite long, so might just be worth a statement saying you agree to the terms and conditions?
